src/bsp/src/stm32/src/interloc/include/interloc/InterlocStateHandler.h

```diff
@@ -51,9 +54,7 @@ class InterlocStateHandler {
     CircularQueue<StateTransition> m_stateTracer;
 
     uint8_t m_sequenceID = 0;
-
-    // TODO : set manually /AprilTag
```

Comment:
How is it handled right now? (Not an issue, just curious)

Reply:
We use the board id (the same that is returned when you greet with the HM). In the next days, we will embed this ID directly in the Nucleo's memory so they will each have a unique ID from 1 to 5